source/_integrations/garadget.markdown

```diff
@@ -127,10 +127,12 @@ For configuration of the garadget as a MQTT cover:
 cover:
   - platform: mqtt
```

Comment:
I think we should remove this example from the integration page. It has nothing to do with this integration. It describes an MQTT integration configuration that the device separately, regardless of this integration, supports.

Reply:
I agree 👍